.readthedocs.yaml

```diff
@@ -18,7 +18,7 @@ mkdocs:
 # Optionally declare the Python requirements required to build your docs
 python:
   install:
-    - requirements: requirements.txt
-    - requirements: docs/requirements.txt
     - method: pip
       path: .
+    - requirements: requirements.txt
```

Comment:
Any reason for this update?

Reply:
yes! The `method: pip path: .` actually overrides all the dependencies and put them to the last version.

So by putting `pip install -r requirements.txt` after, we enfore all the dependencies to be like in the `requirements.txt` file.